### PR TITLE
give opportunity to set zoom lvls

### PR DESCRIPTION
**Pull request by Brynjulf**

### CODE REVIEW

File: src/control/LayerManager.ts [hunk 1]
```diff
@@ -145,6 +145,21 @@ export class LayerManager {
     return this;
   }
 
+  setZoomLevel(zoomlevels: [number, number]): LayerManager {
```
Comment (HavardNJ) [inline src/control/LayerManager.ts:148]: Maybe call this something else like setZoomLevelLimits or setMinMaxZoomLevel, my first thought when I see setZoomLevel is a function I can use to set the current zoom level

File: src/control/ZoomPanHandler.ts [hunk 2]
```diff
@@ -231,7 +234,7 @@ export class ZoomPanHandler {
    * Initialized handler
    */
   init(): void {
-    this.zoom = zoom().scaleExtent([0.1, this.options.maxZoomLevel]).on('zoom', this.onZoom);
+    this.zoom = zoom().scaleExtent([DEFAULT_MIN_ZOOM_LEVEL, this.options.maxZoomLevel]).on('zoom', this.onZoom);
```
Comment (HavardNJ) [inline src/control/ZoomPanHandler.ts:237]: Maybe min zoom level should be possible to set through ZoomAndPanOptions as well?

Comment (Brynjulf) [reply src/control/ZoomPanHandler.ts]: done

File: src/control/LayerManager.ts (hunk 1 above)
Comment (Brynjulf) [reply src/control/LayerManager.ts]: renamed it to setZoomLevelBoundary, hope that will cover it

**Pull request merged**